velox/exec/QuerySplitTracer.h

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Can we come up a more consistent naming for the file like
TraceXXXWriter/Reader? Thanks!